Packages/Tracking/Interaction Engine/Runtime/Scripts/UI/Anchors/AnchorableBehaviour.cs

```diff
@@ -352,12 +367,28 @@ void Awake()
                     interactionBehaviour.OnGraspEnd += tryToAnchorOnGraspEnd;
                 }
             }
-
+            if (_rigidbody == null)
+            {
+                _rigidbody = GetComponent<Rigidbody>();
+            }
             initUnityEvents();
         }
 
         void Start()
         {
+            if (anchor.GetChiralityOfAttachedHand() != ChiralitySelection.NONE)
+            {
+                if (!TryGetComponent<IgnorePhysicalHands>(out _ignorePhysicalHands))
+                {
+                    _ignorePhysicalHands = this.gameObject.AddComponent<IgnorePhysicalHands>();
+                }
```

Comment:
I think we shouldn't force the component on them if they haven't added it themselves - this will also be forcing it if they aren't even using PhysHands